cmd/crossbuild.go

```diff
@@ -125,27 +125,31 @@ func runCrossbuild() {
 
 	if mainPlatformsParam := strings.Join(mainPlatforms[:], " "); mainPlatformsParam != "" {
 		fmt.Println("> running the main builder docker image")
-		sh("docker run --rm -t -v $PWD:/app",
-			dockerMainBuilderImage, "-i", repoPath, "-p", q(mainPlatformsParam))
+		if err := docker("run --rm -t -v $PWD:/app", dockerMainBuilderImage, "-i", repoPath, "-p", q(mainPlatformsParam)); err != nil {
+			fatalMsg("The main builder docker image exited unexpectedly", err)
```

Comment:
Already done by the `fatalMsg()` function. https://github.com/prometheus/promu/blob/master/cmd/promu.go#L132-L135